static/css/siglens.css

```diff
@@ -2468,6 +2468,22 @@ input[type="submit" i] {
     column-gap: 50px;
 }
 
+
+.default-label {
+    color: white;
+    background-color: #364fef;
+    padding: 2px 6px;
+    border-radius: 3px;
+    font-size: 12px;
+    margin-right: 6px;
+    text-decoration: none !important;
+}
+
+
+a {
```

Comment:
Looks good.
nit: Do not directly apply styles to HTML elements. Instead, give them a specific ID or class; otherwise, the styles will apply to all elements present.

Reply:
https://github.com/siglens/siglens/assets/71771131/72ab7666-10e4-4ac9-9554-3248b758b6d7

When we hover over the delete icon it moves forward.

Reply:
The 'a' change was inconsistent, so I removed it. The issue with the moving thing was due to changes in margins, but I've fixed it now!